trillian/ctfe/handlers_test.go

```diff
@@ -996,6 +1015,26 @@ func TestGetProofByHash(t *testing.T) {
 					},
 				},
 			},
+			httpRsp: &inclusionProof,
+		},
+		{
+			// Hash with URL-encoded %2B -> '+'.
+			req:  "hash=WtfX3Axbm7UwtY7GhHoAHPCtXJVrY5vZsH%2ByaXOD2GI=&tree_size=1",
```

Comment:
I think it's supposed to be highlighting syntax errors, but % doesn't need to be escaped in a Go string so it looks like a false positive.